### PR TITLE
tests: hash more files for devtools test cache

### PR DESCRIPTION
**Pull request by connorjclark**
For changes like https://github.com/GoogleChrome/lighthouse/pull/11411 . GitHub actions cache will keep stuff around for a week, so we need a way to invalidate it when the devtools bundle changes.

### CODE REVIEW

**Comment by connorjclark** [comment]
Oh, I do expect this to fail now because of https://github.com/GoogleChrome/lighthouse/issues/11414#issuecomment-691281339

File: .github/workflows/devtools.yml [hunk 1]
```diff
@@ -44,6 +33,17 @@ jobs:
...
+    # Caches last ~week, so CI will refresh these dependencies that often.
+    - name: Cache depot tools, devtools, blink tools and content shell
+      uses: actions/cache@v2
+      with:
+        path: |
+          ${{ env.DEPOT_TOOLS_PATH }}
+          ${{ env.DEVTOOLS_PATH }}
+          ${{ env.BLINK_TOOLS_PATH }}
+          ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
+        key: ${{ runner.os }}-${{ hashFiles('lighthouse/.github/workflows/devtools.yml', 'lighthouse/dist/lighthouse-dt-bundle.js', 'lighthouse/lighthouse-core/test/chromium-web-tests/download-*') }}
```
Comment (brendankenny) [inline .github/workflows/devtools.yml:46]: > `lighthouse/dist/lighthouse-dt-bundle.js`

won't this mean it won't use the cache on basically any change to core?

Comment (connorjclark) [reply .github/workflows/devtools.yml:46]: shoot, good point. I will use the build bundle, devtools entry, and devtools reports files instead.

**Review by brendankenny — APPROVED** [review]
LGTM, probably! 😁

**Comment by paulirish** [comment]
Current CI failure is that is cuz it wants #11418 landed

image

**Comment by paulirish** [comment]
i guess the retry_3 line in run-web-tests.sh is being a jerk

image

**Pull request merged**